CONTRIBUTING.md

```diff
@@ -4,6 +4,19 @@ Pull requests are welcome for any part of this project. Please feel free to send
 
 If you have any thoughts about bigger contributions, please consider [filing an issue](https://github.com/cubing/cubing.js/issues/new) to discuss it before sending a pull request. (Since we're still [below version 1.0](https://semver.org/#spec-item-4), some relevant design choices are still not settled or written down yet.)
 
+## First-Time Contributors
+
+If this is your first time contributing to a project on Github, these are the steps you need to take to contribute code:
+
+1. Create a fork of `cubing.js` 
```

Comment:
Could I ask you to put a period (`.`) at the end of every bullet?

There's nothing wrong with leaving it out, but that will make it more consistent with the rest of the project.

Reply:
Periods added per request!